Assets/Vimeo/Scripts/Player/VimeoPlayer.cs

```diff
@@ -370,5 +395,24 @@ private void ApiError(string response)
                 OnLoadError();
             }
         }
+
+#if VIMEO_DEPTHKIT_SUPPORT
+        public bool HasValidDepthkitMetadata(JSONNode metadata)
```

Comment:
We should delete this detection and let Depthkit just error instead of us trying to catch it. What if params change? We would have to update both Depthkit and Vimeo to resolve it.